### DISCUSSION TITLE
绝区零怎么玩

### DISCUSSION THREAD

**Discussion by mcnb59** [share121/inter-knot#2895] [其他]
《真》不会玩

**Comment by FRIKY-c** [comment]
啊？

**Comment by shiliuyexiafeng** [comment]
简单，我教你，aaa就行了<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by Aha-SWYH** [comment]
很简单，就像玩绝区零那样玩就行了

**Comment by demolute** [comment]
根据绝区零十年清朝老兵描述，zzz睡觉就行了
